keepassxc-browser/background/keepass.js

```diff
@@ -795,7 +795,7 @@ keepass.enableAutomaticReconnect = async function() {
     if (keepass.reconnectLoop === null) {
         keepass.reconnectLoop = setInterval(async () => {
             if (!keepass.isKeePassXCAvailable) {
-                keepass.reconnect();
+                keepass.reconnect(null, 1500); // 1.5 seconds timeout for reconnect
```

Comment:
This change is no longer necessary given the new defaults

Reply:
Yes, you are right. I've rolled back the change.